void tomo_db0::IoLoop::run_loop<false, true, false, true, true>()
--- PRE

+++ POST

@@ -581,7 +581,7 @@

 0afd [ 3] nopl (%rax)
 0b00 [ 5] mov $0x1,%esi
 0b05 [ 3] mov %rbx,%rdi
-0b08 [ 5] call <tomo_db0::IoLoop::arm_accept(tomo_db0::UrKind)>
+0b08 [ 5] call <tomo_db0::IoLoop::fliplisten_retry_accept(tomo_db0::UrKind)>
 0b0d [ 5] jmp <+0x31b>
 0b12 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0b1d [ 3] nopl (%rax)
